### DISCUSSION TITLE
Please join foundation

### DISCUSSION THREAD

**Discussion by dbeavon** [dotnet/spark#1175] [General]
Please join the .Net foundation.

The Membership Form has been updated, here is a link: https://dnf-formlibrary.azurewebsites.net/

The foundation is probably the best way to get this project unblocked.

**Comment by grazy27** [comment]
Done, I guess
![image](https://github.com/user-attachments/assets/cbcb13c7-e2a7-4f30-b7ce-8d3172e29ff3)

**Comment by wudanzy** [comment]
Thanks for confirmation!
